src/redisearch_rs/rqe_iterators/src/id_list.rs

```diff
@@ -78,6 +90,10 @@ impl<'index> IdList<'index> {
         &mut self,
         doc_id: t_docId,
     ) -> Result<Option<(SkipToOutcome<'_, 'index>, usize)>, RQEIteratorError> {
+        if !SORTED && !cfg!(feature = "disable_sort_checks_in_idlist") {
```

Comment:
Is `!cfg!(feature = "disable_sort_checks_in_idlist")` needed here before fixing MOD-12370? Or that we are only calling for `read` today over unsorted IdList iterators?